### ISSUE TITLE
[A11y] Advanced Search - Focus Indicator

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#1804]
### Operating System

Windows

### Browser

Edge

### Affected page

Advanced Search

### Admin user?

Yes

### Issue Description

The facets' mouse click area includes the chevron part, but the keyboard focus doesn't highlight that, making it confusing if you're expecting the focus indicator to surround the entire control.

I don't know if this is a hard failure.  The focus indicator should tell you where your focus is, but the chevron never gets focus.  But the text does get focus.  Is that good enough?  Or does the chevron need it since that's what people see as the control for an accordion / disclosure?  Or does the whole button need a focus indicator around it?

This may be best to hold off since there are other issues with the way these facet disclosures work.

### Suggested Fix

Make the focus indicator surround the whole control if possible

**Comment by jechols** [comment]
Time is *not* on our side.  Ignore this issue unless you find an incredibly cheap fix.  I don't *think* this is a hard WCAG failure.
